packages/core/src/App/Containers/UnlinkPasswordModal/unlink-password-modal.jsx

```diff
@@ -0,0 +1,222 @@
+                                            toggleResetPasswordModal(false);
+                                            logoutClient().then(() => {
+                                                redirectToLogin(false, getLanguage(), false);
+                                            });
+                                        }}
+                                        primary
+                                        large
+                                    >
+                                        <Localize i18n_default_text='Got it' />
+                                    </Button>
+                                </div>
+                            ) : (
+                                <div className='unlink-password__set-password'>
+                                    <Text as='p' weight='bold' className='unlink-password__heading'>
+                                        <Localize i18n_default_text='Deriv password' />
+                                    </Text>
+                                    <Text as='p' size='xs' className='unlink-password__subtext'>
+                                        <Localize i18n_default_text='Enter a new password for your Deriv account.' />
+                                    </Text>
+                                    <fieldset className='unlink-password__input-field'>
+                                        <PasswordMeter
+                                            input={values.password}
+                                            has_error={!!(touched.password && errors.password)}
+                                            custom_feedback_messages={getErrorMessages().password_warnings}
+                                        >
+                                            <PasswordInput
+                                                autoComplete='new-password'
+                                                className='unlink-password__password-field'
+                                                name='password'
+                                                label={localize('Deriv password')}
+                                                onChange={handleChange}
+                                                onBlur={handleBlur}
+                                                error={touched.password && errors.password}
+                                                value={values.password}
+                                                data-lpignore='true'
+                                                required
+                                            />
+                                        </PasswordMeter>
+                                    </fieldset>
+                                    <Text as='p' size='xs' className='unlink-password__hint'>
+                                        {status.error_msg ? (
+                                            <Localize
+                                                i18n_default_text='{{error_msg}}'
```

Comment:
There's only variable here, and no text to localize. There's no need to use `Localize` here.